Sources/MediaViewer/PreviewItemView/Builtins/PlayerPreviewItemViewController.swift

```diff
@@ -1,51 +1,66 @@
 import UIKit
 import AVKit
+import VideoToolbox
 import os
 
 fileprivate let logger = Logger(
     subsystem: Bundle.main.bundleIdentifier! + ".logger",
     category: #file
 )
 
-public final class PlayerPreviewItemViewController: UIViewController {
-    private let playerView = AVPlayerView()
-    
-    let player: AVPlayer
+@MainActor
+public final class PlayerPreviewItemViewController: AVPlayerViewController {
+    private var playerOutput: AVPlayerItemVideoOutput
     
     public init(player: AVPlayer) {
```

Comment:
If this class is given a player, shouldn't the one creating the player also set the output? Why does this class specifically needs to set the output but nothing else?

Reply:
removed. it's from a previous attempt at getting thumbnail.